ts/routes/image-occlusion/shapes/to-cloze.ts

```diff
@@ -170,10 +170,6 @@ function fabricObjectToBaseShapeOrShapes(
         shape.top = newPosition.y;
     }
 
-    if (size == undefined) {
-        size = { width: 0, height: 0 };
-    }
```

Comment:
What motivated this change? Perhaps related to the Svelte 5 update?

Reply:
Size should never be undefined now; I've moved it higher up in the logic:
`export const getBoundingBoxSize = (): Size => {
    const boundingBoxSize = getBoundingBox()?.getBoundingRect(true);
    if (boundingBoxSize) {
        return {width: boundingBoxSize.width, height: boundingBoxSize.height};
    }
    return {width: 0, height: 0};
}`